content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
```

Comment:
@dellintosh I think you must set default branch as `drone-0.5` since drone has been release 0.5 now. http://readme.drone.io/